contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/wifi_test.py

```diff
@@ -0,0 +1,296 @@
+        time.sleep(10)
+    logging.error("Not able to connect to DUT AP SSID {}".format(manager.ssid))
+    sys.exit(1)
+
+
+def ping_test(manager, ip, user, pwd):
+    try:
+        logging.info("Attempting to ping DUT...")
+        ping_result = manager.run_command(
+            sshpass_cmd_gen(ip, user, pwd, ping_cmd(manager.get_ip_addr()))
+        )
+        packet_loss = re.search(r"(\d+)% packet loss", ping_result).group(1)
+        logging.info("Packet loss: %s %%", packet_loss)
+        if packet_loss == "0":
+            logging.info("Ping DUT pass")
+            exit_code = 0
+        else:
+            logging.error("Ping DUT fail with %s %% packet loss!", packet_loss)
+            exit_code = 1
+    except Exception as e:
+        logging.error("An error occurred during ping_test: %s", str(e))
+        exit_code = 1
+    finally:
+        del_host_conn = "{} c delete {}".format(manager._command, manager.ssid)
+        try:
+            manager.run_command(sshpass_cmd_gen(ip, user, pwd, del_host_conn))
+            logging.info("Deleted host connection successfully.")
+        except Exception as e:
+            logging.error("Failed to delete host connection: %s", str(e))
+        sys.exit(exit_code)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="WiFi test")
+
+    parser.add_argument(
+        "--type", required=False, default="wifi", help="Connection type."
+    )
+    parser.add_argument(
+        "--mode",
+        choices=["ap", "adhoc"],
+        required=False,
+        help="WiFi mode: ap or adhoc",
+    )
+    parser.add_argument(
+        "--interface", required=False, help="WiFi interface to use"
+    )
+    parser.add_argument("--band", required=False, help="WiFi band to use")
+    parser.add_argument(
+        "--channel", required=False, help="WiFi channel to use"
+    )
+    parser.add_argument(
+        "--keymgmt", required=False, help="Key management method"
+    )
+    parser.add_argument(
+        "--group", required=False, help="Group key management method"
+    )
+    parser.add_argument("--ssid", required=False, help="SSID for AP mode")
+    parser.add_argument("--ssid-pwd", required=False, help="Password for SSID")
+    parser.add_argument(
+        "--peer", required=False, help="MAC address for p2p peer"
+    )
+    parser.add_argument(
+        "--host-ip",
+        required=False,
+        help="IP address of the Host device to connect to"
+        "The HOST is a device to access the DUT' AP.",
+    )
+    parser.add_argument(
+        "--host-user",
+        required=False,
+        help="Username of the Host device for SSH connection",
+    )
+    parser.add_argument(
+        "--host-pwd",
+        required=False,
+        help="Password of the Host device for SSH connection",
+    )
+    parser.add_argument(
+        "--set-ap-only",
+        action="store_true",
+        help="Setup DUT's AP mode only. Mostly use for debugging.",
+    )
+
+    args = parser.parse_args()
```

Comment:
most arguments are mandatory, please set the `required` to True

Reply:
I've pushed the fix

Reply:
I still observe this issue